lib/pf/scan.pm

```diff
@@ -188,9 +191,11 @@ Prepare the scan attributes, call the engine instantiation and start the scan
 =cut
 
 sub run_scan {
-    my ( $host_ip, $mac ) = @_;
+    my ( $host_ip, $mac , $C_scanner ) = @_;
```

Comment:
$C_scanner doesn't really follow our naming convention and isn't self-explanatory as a variable name

Reply:
Done